duo_api.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name        = 'duo_api'
-  s.version     = '1.0.0'
+  s.version     = '1.0.0.dev'
```

Comment:
It's a convention I've seen in a number of gems. You set the version to 'x.y.z.dev' in between releases. When you're ready to cut the new gem, the maintainer removes the suffix and increments the major/minor/patch numbers as appropriate.

If you'd rather I set it to 1.0.1 so you can cut the gem immediately after merging, no problem!